config/development.toml

```diff
@@ -615,3 +615,6 @@ payment_attempts = "hyperswitch-payment-attempt-events"
 payment_intents = "hyperswitch-payment-intent-events"
 refunds = "hyperswitch-refund-events"
 disputes = "hyperswitch-dispute-events"
+
+[allowed_payment_methods_for_pm_client_secret]
```

Comment:
We should go for a more appropriate name, along with namespacing if necessary. Something like
```
[saved_payment_methods]
eligible_via_sdk = ["card"]
```
for example. Feel free to choose a name as appropriate.